lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -262,7 +262,7 @@ class HttpProtocol extends Protocol {
 
     debug('HTTP request response: %a', data);
 
-    if (data.raw) {
+    if (data.raw && data.content) {
       if (typeof data.content === 'object') {
```

Comment:
```suggestion
      if (data.content && typeof data.content === 'object') {
```

Reply:
not enough: if you do that, you end up stringifying null and returning a string with the word "null" in it

you have to specifically return null if data.content is null, to truly send a 0-length response